PerimeterXModule/Internals/Validators/PXS2SValidator.cs

```diff
@@ -90,9 +90,11 @@ public RiskResponse SendRiskResponse(PxContext PxContext)
                     HttpMethod = PxContext.HttpMethod,
                     HttpVersion = PxContext.HttpVersion,
                     RiskMode = riskMode,
-                    PxCookieHMAC = PxContext.PxCookieHmac
-
-                }
+                    PxCookieHMAC = PxContext.PxCookieHmac,
+					CookieOrigin = PxContext.CookieOrigin,
+					OriginalUUID = PxContext.OriginalUUID,
+					OriginalTokenError = PxContext.OriginalTokenError
```

Comment:
In the case of a regular (non mobile) flow, what will be the values of `OriginalUUID`, `OriginalTokenError`? 
They will all have a default of an empty string?